docs/reference/indices/put-mapping.asciidoc

```diff
@@ -1,24 +1,14 @@
 [[indices-put-mapping]]
 == Put Mapping
 
-The PUT mapping API allows you to add a new type to an existing index, or add new
-fields to an existing type:
+The PUT mapping API allows you to define a mapping for a new or existing index by adding fields to your mapping type.
```

Comment:
good catch, but a small nit - the put mapping api only works for existing indices. How about `The PUT mapping API allows you to add fields to an existing index or to change search only settings of existing fields`?

Reply:
Ah makes sense! Just made the change.